packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.8.0", "<0.9.0"]
+    version: [">=0.8.0", "<0.10.0"]
```

Comment:
I think we can now remove `dbt_utils` as a package dependency entirely! (`dbt_utils.equality` is still used in integration tests, so this file should just move to `integration_tests/packages.yml`)

Reply:
This is actually what I set out to do with the PR. But then I saw references in the PR template and contributor guide and thought it may be too aggressive to yank completely.